.eslintrc

```diff
@@ -0,0 +1,24 @@
+{
+    "rules": {
+        "indent": [
+            2,
+            2
+        ],
+        "quotes": [
+            1,
+            "single"
+        ],
+        "linebreak-style": [
+            2,
+            "unix"
+        ],
+        "semi": [
+            2,
+            "always"
+        ]
+    },
+    "env": {
+        "node": true
+    },
+    "extends": "eslint:recommended"
```

Comment:
Awesome, thanks.

For "no-underscore-dangle" I made it a warning only because we currently have 25 instances of that.

As we discussed on IRC, leaving out "quotes backtick" until the ES6 refactor, as it adds almost 400 lint warnings.

Rule "prefer-template" is for ES6 only, so leaving out for now as it introduces 30 errors.

Rule "prefer-arrow-callback" is for ES6 only and introduces 150 lint errors, so leaving out for now.

Rule "comma-dangle" is already turned on/included in the recommended rules with the default of "never".

Some of the rules don't exist (even after I upgraded to the latest eslint), so these rules are not added:

Definition for rule 'no-empty-pattern' was not found
Definition for rule 'no-magic-numbers' was not found